src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/Validation/OnboardedCountriesAccess.swift

```diff
@@ -6,6 +6,12 @@ import Foundation
 import SwiftCBOR
 
 public struct OnboardedCountriesAccess {
+    
+    // MARK: - Init
+
+    public init() {}
+    
+    // MARK: - Public
```

Comment:
Why is `extractCountryCodes` not static then? This would prevent this initializer.

Reply:
Hm I only added it to be consistent to the rest in the ToolKit. Why all these Access struct does not have static methods: I don't know. @ArturFriesen what was your approach here?

Reply:
Because there is a big chance that we want to have a mock for that and replace it during runtime, for example for testing. And it increases composability.

Reply:
I even think the other way, we should put a protocol on top and inject it into the service. But lets not do it on this PR. There will be others following with more testing which will require this.

Reply:
yeah, sounds like a plan (Protocol + other PR).